services/net/clip/ClipAction.cs

```diff
@@ -478,9 +486,7 @@ private static string GetOtherArgs(IngestModel ingest)
     private static string GetCopy(IngestModel ingest)
     {
         var value = ingest.GetConfigurationValue("copy");
-        // Attempting to use this copy command is much faster, but it corrupts the output file...
-        // return String.IsNullOrWhiteSpace(value) ? " -c:v copy -c:a copy" : $" {value}";
-        return String.IsNullOrWhiteSpace(value) ? "" : $" {value}";
+        return String.IsNullOrWhiteSpace(value) ? " -c:v copy -c:a copy" : $" {value}";
```

Comment:
Put it back the way it was as this is far more efficient.  The real issue appears to be related to the stream missing content.